tools/test.py

```diff
@@ -234,9 +235,15 @@ def main():
         model.CLASSES = dataset.CLASSES
 
     if not distributed:
-        model = build_dp(model, cfg.device, device_ids=cfg.gpu_ids)
-        outputs = single_gpu_test(model, data_loader, args.show, args.show_dir,
-                                  args.show_score_thr)
+        if args.ipu:
+            from mmcv.device.ipu import build_ipu_model
+            model = build_ipu_model(model, **cfg.runner)
+            outputs = ipu_test(model, data_loader, args.show, args.show_dir,
```

Comment:
single_ipu_test

Reply:
it is model parallelism, the model is running on 4 IPUs, how about single_replica_ipu_test

Reply:
OK. But maybe add a note here.